src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +58,29 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        final MovieSessionService movieSessionService =
+                (MovieSessionService) INJECTOR.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        final AuthenticationService authenticationService =
+                (AuthenticationService) INJECTOR.getInstance(
+                        AuthenticationService.class);
+        User bob = new User(BOB_EMAIL, BOB_PASSWORD);
+        User alice = new User(ALICE_EMAIL, ALICE_PASSWORD);
+        User unknown = new User(BOB_EMAIL, ANOTHER_PASSWORD);
+
+        System.out.println(
+                authenticationService.register(bob.getEmail(), bob.getPassword()));
+        System.out.println(
+                 authenticationService.login(bob.getEmail(), bob.getPassword()));
+
+        // authenticationService.register(unknown.getEmail(), unknown.getPassword());
+        // authenticationService.login(alice.getEmail(), alice.getPassword());
+        // authenticationService.login(bob.getEmail(), ANOTHER_PASSWORD);
```

Comment:
These lines are commented out, which suggests that the functionality for registering and logging in with different users is not fully tested or implemented. Consider uncommenting and testing these lines to ensure that the authentication service works as expected.